src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -573,13 +578,15 @@ public partial interface ISecurityPolicyEncodable
         void FromXml(System.Security.SecurityElement e, System.Security.Policy.PolicyLevel level);
         System.Security.SecurityElement ToXml(System.Security.Policy.PolicyLevel level);
     }
+#if netstandard || netfx
```

Comment:
You're missing net461 which is actually the assembly carried over in the package for full framework. So you would either need to do a `!netcoreapp && !uap` or add a DefineConstants entry for net461 here: https://github.com/dotnet/corefx/blob/master/Directory.Build.props#L418

 then just do `netstandard || netfx || net461`